### DISCUSSION TITLE
What is the ordering output from the "to_cartesian" method?

### DISCUSSION THREAD

**Discussion by JoshRackers** [e3nn/e3nn#378] [General]
The `CartesianTensor` class has a very useful method, `to_cartesian`. (Docs here: [https://docs.e3nn.org/en/stable/api/io/cartesian_tensor.html](url)) What is the xyz ordering of the output of this method? Specifically, what is the ordering for L>1 tensors? 

I suppose this also applies to the `from_cartesian` method as well...

**Comment by mariogeiger** [comment]
e3nn library uses the same irreps everywhere, so the ordering for the output of the `to_cartesian` method for L>1 tensors is `[x,y,z]`. This is also why the spherical harmonics for L=1 is proportional to the identity in e3nn, as all vectors in the library are represented in this ordering.

**Reply by JoshRackers** [reply]
That helps- thanks!
